pages/attacks/Content_Security_Policy.md

```diff
@@ -309,7 +309,8 @@ Firefox (Mozilla CSP directives).*
 
 ## Tools
 
-[w3af](Automated_Audit_using_w3af "wikilink") audit tools
+There's a number of tools that can assist with the generating, evaluation and monitoring of content security policy:
+* [w3af](Automated_Audit_using_w3af "wikilink") audit tools
```

Comment:
Could you set this to a proper current link for the tool?

Reply:
Changed to the full link